### ISSUE TITLE
Error: can't decode byte 0xc0 in position 0

### ISSUE THREAD

**Issue by Ghxst** [frida/frida-java-bridge#129]
Hey, this error is popping up while using Dwarf. 
I was advised by Gio to open an issue for it here, since it's related to Frida.

```
payload: {'type': 'error', 'description': "Error: can't decode byte 0xc0 in position 0", 'stack': "Error: can't decode byte 0xc0 in position 0\n    at NativePointer.readUtf8String (frida/runtime/core.js:144:1)\n    at Env.stringFromJni (frida/node_modules/frida-java/lib/env.js:930:1)\n    at klass.fromJni (frida/node_modules/frida-java/lib/class-factory.js:2435:1)\n    at klass.f (eval at makeMethod (frida/node_modules/frida-java/lib/class-factory.js:1491:1), <anonymous>:1:587)\n    at klass.f (frida/node_modules/frida-java/lib/class-factory.js:1318:1)\n    at klass.<anonymous> (/script1.js:2196:43)\n    at f (eval at implement (frida/node_modules/frida-java/lib/class-factory.js:2029:1), <anonymous>:1:293)", 'fileName': 'frida/runtime/core.js', 'lineNumber': 144, 'columnNumber': 1}
```

I have attached the full log (it's big).
[output.txt](https://github.com/frida/frida-java/files/3467883/output.txt)

This is happening consistently but not always at the same time, I will run some more tries and update the issue with more error logs.

**Comment by ghxstdev** [comment]
[output.txt](https://github.com/frida/frida-java/files/3468026/output.txt)

**Comment by ghxstdev** [comment]
[output.txt](https://github.com/frida/frida-java/files/3468031/output.txt)

**Comment by ghxstdev** [comment]
Last log.
[output.txt](https://github.com/frida/frida-java/files/3468071/output.txt)

**Comment by Alexeyan** [comment]
I have the same issue. Looks like the String-conversion from UTF-16(Java-Strings) to UTF-8(Javascript-Strings) does not work properly.

**Comment by Ghxst** [comment]
@Alexeyan I think you are correct, this seems to be an issue with encoding. I think it's a python thing 😅.

**Comment by oleavr** [comment]
Fixed in #164 – releasing this fix now as part of 12.9.5.
